Sources/ArcGISToolkit/Components/Form/FormInputs/RadioButtonsInput.swift

```diff
@@ -0,0 +1,170 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct RadioButtonsInput: View {
+    @Environment(\.formElementPadding) var elementPadding
+    
+    /// The set of options in the input.
+    @State private var codedValues = [CodedValue]()
+    
+    /// A Boolean value indicating whether the date selection was cleared when a value is required.
+    @State private var requiredValueMissing = false
+    
+    /// The selected option.
+    @State private var selectedValue: CodedValue?
+    
+    /// A Boolean value indicating whether the current value doesn't exist as an option in the domain.
+    ///
+    /// In this scenario a ``ComboBoxInput`` should be used instead.
```

Comment:
```suggestion
    /// A Boolean value indicating whether a `ComboBoxInput`` should be used instead. This will be `true` if
    /// the current value doesn't exist as an option in the domain
```